lighthouse-core/audits/network-requests.js

```diff
@@ -77,6 +78,8 @@ class NetworkRequests extends Audit {
         {key: 'resourceType', itemType: 'text', text: 'Resource Type'},
       ];
 
+      // TODO(exterkamp): fix ts here
+      // @ts-ignore
```

Comment:
@brendankenny ideas here?

Reply:
discussed offline: split up `lrStatistics` into its individual properties. If this wasn't a table it would be fine to have it as an object, but we shouldn't be putting objects into table entries.

The other solution is to note that this isn't actually a table, just a hidden diagnostic that's an array of sets of key:value pairs calling itself a table, and switch to a "diagnostic" audit details type, which can include anything. That'll have to wait for 5.0 though, if we really want this as an object.

Reply:
Fixed.